### PR TITLE
Some wording improvements, clarifications, etc.

### PR DESCRIPTION
**Pull request by ThomasLandauer**

### CODE REVIEW

**Comment by ThomasLandauer** [comment]
Question: I'm not sure about this line:

```javascript
newForm = newForm.replace(/__name__label__/g, index);
```

I can never see `__name__label__` in my prototype (Symfony 4.1), no matter if I set `'entry_options' => array('label' => false)` or not. Was this feature maybe removed?

And I don't even see what its point would have been. Put the index inside the label? Why?

**Comment by ThomasLandauer** [comment]
Started with some minor corrections, ended up with a major reorganization ;-)

I'm done for now, please merge.

**Comment by weaverryan** [comment]
Hey @ThomasLandauer!

This is awesome! Sorry for the delay - these big changes are hard to review. Do you have a test repository where you built the code? We really need to code through this to make sure everything still works properly.

Cheers!

**Comment by ThomasLandauer** [comment]
> Do you have a test repository where you built the code?

No, I entered everything on GitHub's website. But I didn't change the actual code that much - just moved stuff around ;-)

**Comment by weaverryan** [comment]
Ah, ok! Were there any parts of the code that you changed in any significant way? The diff in these re-org PR's is hard to check (I'm the king of massively re-organizing stuff, so that's not a complaint about your PR!)

**Comment by ThomasLandauer** [comment]
To be honest: I don't really remember :-( But I guess not, I think I just explained things better.

File: form/form_collections.rst [hunk 1]
```diff
@@ -245,47 +245,60 @@ Allowing "new" Tags with the "Prototype"
...
 
-In addition to telling the field to accept any number of submitted objects, the
-``allow_add`` also makes a *"prototype"* variable available to you. This "prototype"
-is a little "template" that contains all the HTML to be able to render any
-new "tag" forms. To render it, make the following change to your template:
+In addition to telling the field to accept any number of submitted objects,
+``allow_add`` also makes a ``prototype`` variable available to you. This
+"prototype" is a little "template" that contains all the HTML needed to dynamically
+render any new "tag" forms with JavaScript. To render the prototype, make the following
+change to the existing ``<ul>`` in your template::
```
[the remark this answers is not in the document]
Comment (ThomasLandauer) [reply form/form_collections.rst]: OK, done :-)

File: form/form_collections.rst [hunk 2]
```diff
@@ -572,60 +555,57 @@ Now, you need to put some code into the ``removeTag()`` method of ``Task``::
...
-~~~~~~~~~~~~~~~~~~~~~~
-
 The ``allow_delete`` option means that if an item of a collection
 isn't sent on submission, the related data is removed from the collection
 on the server. In order for this to work in an HTML form, you must remove
 the DOM element for the collection item to be removed, before submitting
 the form.
 
-First, add a "delete this tag" link to each tag form:
+First, prepare a JavaScript function that adds a "delete this tag" link::
```
Comment (xabbuh) [inline form/form_collections.rst]: oh, here we have  the same :)

Comment (ThomasLandauer) [reply form/form_collections.rst]: Maybe somewhere else too, cause I never understood the principle...

**Comment by ThomasLandauer** [comment]
Do you want me to set up a repository as explained at https://symfony.com/doc/current/contributing/code/reproducer.html ?

**Comment by OskarStark** [comment]
Hi @ThomasLandauer 👋 

> Do you want me to set up a repository as explained at 

That would be cool!

**Comment by OskarStark** [comment]
Hey Thomas 😊

We really appreciate your work, but the 2.8 branch isn’t maintained anymore. Please rebase on 3.4 branch first, most of the things are already fixed and we don’t want to waste your time.

Anyway thanks for your help 👌🏻

Cheers, Oskar

**Comment by ThomasLandauer** [comment]
See https://github.com/ThomasLandauer/symfony-docs_pr_10180

Sorry, I don't know how to rebase :-(
And I must say: I finished this over a year (!) ago, but nobody merged it. And you told me just 5 weeks ago to create the repo. So I've done that.

So please help me out now, I spent hours for this!

**Comment by ThomasLandauer** [comment]
Suggestion: Separate headings for all possible relationship types.

Right now, the page contains too many "In case you have A, you need to pay attention to B, etc.". In this phase (i.e. when the user thinks about setting up an "add" button"), the database model is fixed anyway! Nobody will say "Well, if it's easier to get the button, let's change the relationship type!"

So wouldn't it be better to take the relationship type as granted and give a step-by-step tutorial for each type?

**Comment by OskarStark** [comment]
Hi Thomas 👋 

I am sorry, but I will to close here, `2.8` isn't maintained for a long time.
We really appreciate your work on the docs, but I would recommend to make more smaller changes instead of one of such size.

Thank you for your understanding!

Cheers, Oskar

**Comment by ThomasLandauer** [comment]
July 30:
> > Do you want me to set up a repository as explained at
> 
> That would be cool!

November 14:
> I am sorry, but I will to close here

**Comment by OskarStark** [comment]
Hi Thomas,

I am sorry but you offered todo this and I just wanted to say "yes that would be great and more easy for the one who will help you", so I saw my position more or less to be a moderator on topics/issues/pull requests. Sorry for being not clear enough and you thought, that I could help you out.
Maybe I mixed things up  (because there are so much issues/pull requests I am working on) as a reproducer is mainly helpful for a code issue, but not for docs.

My recommendation is still to provide small pull requests, your last one was merged really quickly (grammar, typo IIRC).

Cheers, Oskar

**Comment by ThomasLandauer** [comment]
I don't know who of you is in charge of what! This isn't documented anywhere (or is it?) So if *somebody* tells me to do something, I usually do it. Oskar, I can't know which role you're playing in the entire game. What I'm blaming you, personally, was to tell me to put more work into this, at a time when it was (I'm 99% sure about this) already futile.

The same is eventually going to happen to https://github.com/symfony/symfony-docs/pull/11222 and https://github.com/symfony/symfony-docs/pull/11223 :-(

Small PR's:
By fixing typos you will never get great documentation. You will get mediocre documentation without typos.  
So if it's hard to review large changes, you need to come up with a better workflow/solution/whatever! Cause right now, you're essentially saying: "Go find some errors, but leave the important stuff up to us!"

**Comment by OskarStark** [comment]
> The same is eventually going to happen to #11222 and #11223 :-(

Eventually.. yes.. 😞 

> By fixing typos you will never get great documentation.

Sorry but many users are very very happy with the Symfony documentation, so personally I am convinced, that we are doing good work.
You have your opinion and I have my opinion, this is be fine and should allowed 👍 

And please remember we are doing all this beside our main jobs ❗️ without any payment ❗️ in our freetime ❗️

Have a good day ☀️ 
oskar

**Pull request closed without merging**